core/battleeffect/actioncard.ts

```diff
@@ -25,7 +25,7 @@ export function getActioncards() {
     experimentalBattlestation,
     fighterPrototype,
     fireTeam,
-    maneuveringJets,
+    // maneuveringJets,
```

Comment:
Commented out, because it's not implemented and is listed as missing in the help text.